src/wazuh_modules/vulnerability_detector/wm_vuln_detector.h

```diff
@@ -922,8 +917,20 @@ void wm_vuldet_free_nvd_node(nvd_vulnerability *data);
 void wm_vuldet_free_nvd_list(nvd_vulnerability *nvd_it);
 const char *wm_vuldet_get_unified_severity(char *severity);
 
-// Initialize tasks queue
-int wm_vuldet_init_queue();
+/**
+ * @brief Initializes global tasks and messages queue.
+ * Vulnerability detector thread finalizes execution on error.
+ */
+void wm_vuldet_init_queues();
+
+/**
+ * @brief Creates a task context to be scheduled in the task queue.
+ *
+ * @param task_parameters
+ * @param ...
```

Comment:
The description of the method isn't enough to understand how to use it without looking at the code.

Reply:
I changed the behavior and added a detailed description.